types/index-test.ts

```diff
@@ -0,0 +1,11 @@
+import parser = require('../');
```

Comment:
IMHO, tests for `.d.ts` files is very strange, because you must install TS and run compilation process. I propose to delete it, sorry (//cc @GitScrum).